src/traces/histogram/attributes.js

```diff
@@ -82,15 +82,21 @@ module.exports = {
         role: 'style',
         description: [
             'Determines whether or not the x axis bin attributes are picked',
-            'by an algorithm.'
-        ].join(' ')
+            'by an algorithm. Note that this should be set to false if you',
+            'want to manually set the number of bins using the attributes in',
+            'xbins.'
+          ].join(' ')
     },
     nbinsx: {
         valType: 'integer',
         min: 0,
         dflt: 0,
         role: 'style',
-        description: 'Sets the number of x axis bins.'
+        description: [
+          'Specifies the maximum number of desired bins. This value will be used',
```

Comment:
we use 4 space indentation.

Run `npm run lint` to check that your patch passes our linting rules.